### PR TITLE
Add table of pangeo deployments 

### LINKED ISSUE #232: List of current Pangeo deployments
We'd like to develop a current listing of Pangeo deployments. If you have deployed pangeo in one form or another, please speak up!

Information we'd like to know:

1. Cloud or HPC (or other)? Which system (e.g. Google Cloud, NCAR's Cheyenne)?
1. How are you deploying dask distributed (KubeCluster, dask-jobqueue, dask-mpi, etc.)?
1. How are you deploying Jupyter (Jupyterhub, single user)
1. Primary use (Tutorial, individual research, etc.)?

(I'll provide a few examples below, sharing a url to your jupyterhub deployment is not required).

xref #229 

---------

**pangeo.pydata.org** (Pangeo EarthCube)
1. Platform: Google cloud
1. Dask: KubeCluster
1. Jupyter: JupyterHub
1. Use: Exploratory deployment for Pangeo EarthCube project. Used for demos, tutorials, research.

**pangeo-aws.cloudmaven.org** (University of Washington)
*Note: this deployment will be taken down soon*
1. Platform: Amazon cloud
1. Dask:KubeCluster
1. Jupyter: JupyterHub
1. Use: Demo deployment / proof of concept for proposal.

**Cheyenne, Caldera, and Geyser** (NCAR)
1. Platform:  HPC 
1. Dask: dask-jobqueue and dask-mpi
1. Jupyter: Single user notebook servers (jupyterhub coming soon: #26 )
1. Use: Individual research

### PR DESCRIPTION
**Pull request by jhamman**
closes #232 

Nothing too fancy here but this gives us something to build on. I've only included deployments listed in #232 but I know there are more. 

deployments

### CODE REVIEW

**Comment by mrocklin** [comment]
Nice

**Comment by jacobtomlinson** [comment]
I've popped ours onto the end of the list! Hope you don't mind me committing on your PR.

**Comment by rabernat** [comment]
Fantastic Joe!

File: docs/_templates/deployments.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
+<h2> Cloud Computing Deployments </h2>
```
Comment (rabernat) [inline docs/_templates/deployments.html]: One issue with hard coding this as HTML is that Sphinx doesn't parse the headings and put them in the table of contents. Can you think of a way around that?

Comment (jhamman) [reply docs/_templates/deployments.html]: I noticed this. I didn't find a clean solution yet and didn't feel like it was a deal breaker. At first I wasn't splitting the list by cloud/hpc so it was even less of an issue.

Comment (rabernat) [reply docs/_templates/deployments.html]: Ok nbd. Probably not worth the effort it would take to figure this out.

**Pull request merged**